jitar/src/server/utils/RuntimeConfigurator.ts

```diff
@@ -46,14 +46,15 @@ export default class RuntimeConfigurator
     {
         const sourceLocation = configuration.source ?? RuntimeDefaults.SOURCE;
         const cacheLocation = configuration.cache ?? RuntimeDefaults.CACHE;
+        const assets = configuration.assets;
```

Comment:
Suggestion for a more descriptive name: `assetFilePatterns`

Reply:
done